### PR TITLE
Avatar and alert component

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added avatar and alert component.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,185 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.0/css/all.min.css"
+      integrity="sha512-BnbUDfEUfV0Slx6TunuB042k9tuKe3xrD6q4mg5Ed72LTgzDIcLPxg6yI2gcMFRyomt+yJJxE+zJwNmxki6/RA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
     <title>Document</title>
   </head>
-  <body></body>
+  <body style="margin: 1rem">
```
Comment (partha8) [inline index.html:26]: you can create a utility class and use it here instead of using inline styles.

File: index.html [hunk 2]
```diff
@@ -4,7 +4,185 @@
...
+      <hr />
+
+      <!-- AVATAR COMPONENT -->
+      <h1>Avatar Component</h1>
+
+      <h3>Avatar</h3>
+
+      <img
+        src="https://www.industrialempathy.com/img/remote/ZiClJf-1920w.jpg"
+        alt=""
```
Comment (partha8) [inline index.html:90]: Add alt, as it is good practice

File: index.html [hunk 3]
```diff
@@ -4,7 +4,185 @@
...
+        This is a Danger Outline Alert<i class="fas fa-times"></i>
+      </div>
+
+      <div class="alert alert-outline-warning">
+        This is a Warning Outline Alert<i class="fas fa-times"></i>
+      </div>
+
+      <br />
+      <br />
+      <br />
```
Comment (anik31) [inline index.html:80]: Don't use so many line breaks. You can add spacing.

File: index.html (hunk 1 above)
Comment (ParthParolekar) [reply index.html:26]: I have a utility class but it slipped form my mind to use it. Thanks for pointing it out.

File: index.html (hunk 2 above)
Comment (ParthParolekar) [reply index.html:90]: Will do. Thank you again.

File: index.html (hunk 3 above)
Comment (ParthParolekar) [reply index.html:80]: Thanks for the feedback. I will do it right away.

**Pull request closed without merging**